2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_setup.py:_flush():71] setting env: {}
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_init.py:_log_setup():371] Logging user logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211202_225201-1w8slt4w/logs/debug.log
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_init.py:_log_setup():372] Logging internal logs to /Users/shy/Library/Mobile Documents/com~apple~CloudDocs/my python programming projects/DTI_pro/wandb/run-20211202_225201-1w8slt4w/logs/debug-internal.log
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_init.py:init():404] calling init triggers
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'max_epochs': 150, 'batch_size': 100}
2021-12-02 22:52:01,069 INFO    MainThread:40638 [wandb_init.py:init():449] starting backend
2021-12-02 22:52:01,069 INFO    MainThread:40638 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-02 22:52:01,081 INFO    MainThread:40638 [backend.py:ensure_launched():199] starting backend process...
2021-12-02 22:52:01,089 INFO    MainThread:40638 [backend.py:ensure_launched():205] started backend process with pid: 40660
2021-12-02 22:52:01,096 INFO    MainThread:40638 [wandb_init.py:init():458] backend started and connected
2021-12-02 22:52:01,098 INFO    MainThread:40638 [wandb_init.py:init():519] updated telemetry
2021-12-02 22:52:01,110 INFO    MainThread:40638 [wandb_init.py:init():549] communicating current version
2021-12-02 22:52:02,430 INFO    MainThread:40638 [wandb_init.py:init():554] got version response 
2021-12-02 22:52:02,430 INFO    MainThread:40638 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-02 22:52:02,669 INFO    MainThread:40638 [wandb_init.py:init():592] starting run threads in backend
2021-12-02 22:52:05,294 INFO    MainThread:40638 [wandb_run.py:_console_start():1816] atexit reg
2021-12-02 22:52:05,295 INFO    MainThread:40638 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-12-02 22:52:05,295 INFO    MainThread:40638 [wandb_run.py:_redirect():1695] Redirecting console.
2021-12-02 22:52:05,297 INFO    MainThread:40638 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-02 22:52:05,297 INFO    MainThread:40638 [wandb_init.py:init():619] run started, returning control to user process
2021-12-02 22:52:05,298 INFO    MainThread:40638 [wandb_watch.py:watch():43] Watching
2021-12-02 22:52:05,304 INFO    MainThread:40638 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-02 22:52:05,304 INFO    MainThread:40638 [wandb_run.py:_restore():1758] restore
2021-12-02 22:52:07,822 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-02 22:52:08,066 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 804
}

2021-12-02 22:52:08,703 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7346
}

2021-12-02 22:52:08,809 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7346
}

2021-12-02 22:52:08,914 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 804
  total_bytes: 7346
}

2021-12-02 22:52:09,019 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:09,121 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:09,226 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:09,331 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:09,436 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:09,954 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}

2021-12-02 22:52:10,573 INFO    MainThread:40638 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7346
  total_bytes: 7346
}
local_info {
}

2021-12-02 22:52:11,878 INFO    MainThread:40638 [wandb_run.py:_append_files():2184] logging synced files
